special/OreDictEntryManager.php

```diff
@@ -99,9 +116,20 @@ private function createEntry(FormOptions $opts) {
 			return -2;
 		}
 
-		return OreDict::addEntry($mod, $item, $tag, $this->getUser(), $params);
+		$result = OreDict::addEntry($mod, $item, $tag, $this->getUser(), $params);
+		if ($result == false) {
+			return -2;
+		}
+		return $result;
```

Comment:
Come to think about it, in Java the syntax would be !$results although I suspect that's not a possible thing in PHP.